src/main/app/forms/models/localDate.ts

```diff
@@ -3,14 +3,15 @@ import * as _ from 'lodash'
 import * as moment from 'moment'
 
 import { DATE_FORMAT } from 'utils/momentFormatter'
+import { LocalDate as ValidatorDate } from '@hmcts/cmc-validators'
 
 export class ValidationErrors {
   static readonly YEAR_NOT_VALID: string = 'Enter a valid year'
   static readonly MONTH_NOT_VALID: string = 'Enter a valid month'
   static readonly DAY_NOT_VALID: string = 'Enter a valid day'
 }
 
-export class LocalDate {
+export class LocalDate implements ValidatorDate {
```

Comment:
@ellie-harrison we do not need to say what interface is implemented. The fact that something looks like ValidatorDate is good enough for TS. Please remove.

We introduced `LocalDate` in '@hmcts/cmc-validators' to have some type checking there.